HeartbeatLogging/Tests/Unit/StorageTests.swift

```diff
@@ -100,14 +100,7 @@ class UserDefaultsStorageTests: XCTestCase {
   let suiteName = #file
 
   override func setUpWithError() throws {
-    // Clean up suite in case prior test case did not complete.
-    UserDefaults.standard.removePersistentDomain(forName: suiteName)
-
-    defaults = try XCTUnwrap(UserDefaults(suiteName: suiteName))
-  }
-
-  override func tearDownWithError() throws {
-    defaults.removePersistentDomain(forName: suiteName)
+    defaults = try XCTUnwrap(UserDefaultsFake(suiteName: suiteName))
```

Comment:
Using the real user defaults generates a plist file in the test suite that needs to be manually removed before committing changes. Deleting the file in the class tear down did not work so I added a fake user defaults to inject.